qcodes/instrument_drivers/rigol/DG1062.py

```diff
@@ -0,0 +1,345 @@
+import logging
+from functools import partial
+from typing import Union, Dict
+
+from qcodes import VisaInstrument, validators as vals
+from qcodes import InstrumentChannel, ChannelList
+
+log = logging.getLogger(__name__)
+
+
+def sane_partial(func, docstring, **kwargs):
```

Comment:
shall this reside in "helpers" somewhere?

Reply:
Yeah that could be helpful. Where exactly would you propose to put this?

Reply:
`qcodes/utils/helpers.py` should be an OK place. Although that file is already huge...